examples/cli/discovery.md

```diff
@@ -49,34 +51,14 @@ You can see that the model is discovered, and some stats are printed, but no bui
 
 > See the [Analyze Only documentation](https://github.com/groq/mlagility/blob/main/tools_user_guide.md#analyze-only) for more details.
 
-## Search Directory
+## Benchmark Multiple Scripts
```

Comment:
I think you're missing tutorials/documentation for the full set of features, such as 
`benchit my_model[1-2].py`
`benchit my_model1.py my_model2.py`
`benchit models/*/*.py`

Reply:
Expanded!